src/stylesheets/notification.less

```diff
@@ -6,14 +6,8 @@
 
     .sk-notification {
         overflow: hidden;
-
-        height: @notification-height;
```

Comment:
`@notification-height` is still used, but `@long-notification-height` isn't. I'll remove it.